README.md

```diff
@@ -35,6 +35,23 @@ You will also need to hash your users' passwords using the new hashing algorithm
 
 ## Client-side usage
 
+### Config
+
+```
+var mediator = require('fh-wfm-mediator');
+
+var config = {
+    forceSessionVerificationOnResume: true
+};
+
+userCore(mediator, config);
+
+```
+
+#### forceSessionVerificationOnResume
```

Comment:
That sounds reasonable. Note that event handlers can be stacked together by building chain. addEventHandler("resume") can trigger our function which prevents from bug and user function with additional handlers etc.